packages/nodes-base/nodes/Oracle/Sql/transport/index.ts

```diff
@@ -0,0 +1,66 @@
+import type {
+	IExecuteFunctions,
+	ICredentialTestFunctions,
+	ILoadOptionsFunctions,
+	ITriggerFunctions,
+} from 'n8n-workflow';
+import oracledb from 'oracledb';
+
+import { ConnectionPoolManager } from '@utils/connection-pool-manager';
+
+import type { OracleDBNodeOptions, OracleDBNodeCredentials } from '../helpers/interfaces';
+
+// used for thick mode to call initOracleClient API only once.
+let initializeDriverMode = false;
+
+const getOracleDBConfig = (
+	credentials: OracleDBNodeCredentials,
+	_options: OracleDBNodeOptions = {},
+) => {
+	const { useThickMode, useSSL, ...dbConfig } = credentials;
+	return dbConfig;
+};
+
+export async function configureOracleDB(
+	this: IExecuteFunctions | ICredentialTestFunctions | ILoadOptionsFunctions | ITriggerFunctions,
+	credentials: OracleDBNodeCredentials,
+	options: OracleDBNodeOptions = {},
+): Promise<oracledb.Pool> {
+	const poolManager = ConnectionPoolManager.getInstance(this.logger);
+	const fallBackHandler = async (abortController: AbortController): Promise<oracledb.Pool> => {
+		const dbConfig = getOracleDBConfig(credentials, options);
```

Comment:
If unused param is removed, adjust the call here too
```suggestion
		const dbConfig = getOracleDBConfig(credentials);
```